fili-core/src/main/java/com/yahoo/bard/webservice/web/PreResponse.java

```diff
@@ -34,6 +34,17 @@ public ResponseContext getResponseContext() {
         return responseContext;
     }
 
+    /**
+     * Method to have new PreResponse for the given ResultSet.
```

Comment:
Can we rephrase this to 'Create a new instance of PreResponse using the supplied ResultSet'